spec/models/location_spec.rb

```diff
@@ -34,4 +34,6 @@
     location.destroy!
     expect(volunteer.reload.university_location).to be_nil
   end
+
+  it { is_expected.to have_many(:assembled_boxes).class_name('Box') }
```

Comment:
Here you could also check for `foreign_key` and `inverse_of` attributes.

Reply:
:+1: Thanks!